src/components/cc-article-list/cc-article-list.smart.js

```diff
@@ -47,5 +58,10 @@ async function fetchArticleList ({ signal, lang, limit = 9 }) {
     FOUR_HOURS,
     () => request(requestParams));
 
-  return parseRssFeed(rssFeed, limit);
+  const rawArticleListData = parseRssFeed(rssFeed, limit);
+
+  return rawArticleListData.map((rawArticle) => ({
+    type: 'loaded',
```

Comment:
thought: Okay, I think this line makes me question having a state for `cc-article-card`. My "problem" is that `cc-article-card` is not loading data directly, it will be the role of this smart for `cc-article-list`. It feels like managing two states here. Plus, as we'll only use the card component within the list it might be overkill :thinking:. Feel free to ping me if it's unclear for you. :wink: